dom.bs

```diff
@@ -3398,6 +3400,19 @@ dictionary MutationObserverInit {
    <a>attribute</a> mutations need to be
    observed and {{MutationObserverInit/attributes}} is true
    or omitted.
+
+   <dt>{{MutationObserverInit/elementFilterByAttribute}}
+   <dd>Set to a list of <a>attribute</a>
+   <a for=Attr>local names</a> (without <a for=Attr>namespace</a>) to observe changes
+   to the presence of an elements <a>attributes</a>.
+
+   <dt>{{MutationObserverInit/elementLocalNameFilter}}
+   <dd>Set to a list of <a>element</a>
```

Comment:
You want `<a for=/>element</a>` since there are multiple element definitions.

Reply:
> @AndrewRot can Shape Security sign https://participate.whatwg.org/agreement? Previously that ended up not happening, but not sure what the reason was.

I have signed the participation agreement on behalf of [shapesecurity](https://github.com/shapesecurity) however, the Participation below still states: Pending — @AndrewRot participates on behalf of an unverified entity

Is there something that needs to be verified on behalf of whatwg?